api/autopilot.go

```diff
@@ -121,6 +121,7 @@ type (
 				Pruning  uint64 `json:"pruning"`
 				Upload   uint64 `json:"upload"`
 			} `json:"gouging"`
+			LowMaxDuration        uint64 `json:"lowMaxDuration"`
```

Comment:
Turning this into a Usability check rather than a gouging check simplified things quite a bit and since it is only relevant when forming a contract or renewing it having it as a gouging check might be a bit overkill anyway.